ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse-4.14.yaml

```diff
@@ -79,11 +79,11 @@ tests:
   steps:
     cluster_profile: ibmcloud-qe
     workflow: cucushift-installer-rehearse-ibmcloud-ipi-proxy-private
-- as: installer-rehearse-vsphere-dis
+- as: installer-rehearse-vsphere-ipi-template
```

Comment:
I suggest to add one more connected vsphere job instead of updating the existing disconnected one.

Reply:
@WenXinWei I think you forgot this comment.

Reply:
or you can directly drop the job, we can directly use `vsphere-ipi-template-f14` for testing.